code/datums/holocall.dm

```diff
@@ -736,9 +768,9 @@
 	DELAY 40
 	SAY The first time you witness a breached Abnormality, you will take sanity damage based on the difference between its threat level and your Agent Level.
 	DELAY 55
-	SAY Abnormalities also have varying resistances to each damage type. Read their records to take advantage of this during suppressions.
-	DELAY 45
-	SAY <span class='warning'>It is important you do not move or grab items while working.</span> Otherwise you will automatically fail the work and take heavy damage.
+	SAY Abnormalities will deal a type of damage and have varying resistances to each damage type. Read their records to take advantage of this during suppressions.
+	DELAY 50
+	SAY <span class='warning'>Remember to not move or grab items while working.</span> Otherwise you will automatically fail the work and take heavy damage.
 	DELAY 50
```

Comment:
```suggestion
	SAY <span class='warning'>Leaving the console of an abnormality will cause you to stop working on it.</span> Doing so will cause the work to be labelled a failure and deal damage to you proportional to how much work was left.
	DELAY 50
	SAY If need be, this may serve as an emergency measure should you find yourself in immediate danger due to external threats.
	DELAY 25
```